doc/ChangeLog

```diff
@@ -1,4 +1,132 @@
 ===============================================================
+Tag name:  ctsm5.1.dev022
+Originator(s):  gregorylemieux (Gregory Lemieux,LBL/NGEET,510-486-5049)
+Date:  Wed 27 Jan 2021 12:26:44 AM PST
+One-line Summary: Merge fates_main_api into ctsm master
+
+Purpose and description of changes
+----------------------------------
+
+This brings fates_main_api into and up-to-date with ctsm master.
+At the point of merge, fates_main_api was parallel with ctsm5.1.dev020,
+only one tag behind ctsm master's head.
+
+The purpose of this merge is to make all future updates to fates
+on the ctsm master branch instead of on the fates_main_api branch.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_1
+
+[X] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[X] clm4_5
+
+
+Bugs fixed or introduced
+------------------------
+[Remove any lines that don't apply. Remove entire section if nothing applies.]
```

Comment:
Go ahead and remove these lines that don't apply.

Reply:
Updated per b1a6080a90fa50cc5d9f78efcf2fccc221744cb3